src/simulate/config/config.yml

```diff
@@ -0,0 +1,32 @@
+http:
+  url: http://lti.tools/caliper/event?key=udp_dev
+  token: uy76fuo0gDRT
+users:
+    - id  : user001
+      sis_id : user001_sisid
+      canvas_id : user001_cid
```

Comment:
Where are `sis_id` and `canvas_id` used?  Are they here for use in the next step of development?

Reply:
I have added it now. These id's comes as part of the `federatedSession` object and it was still unclear at the time of development what will go in the fed_session of caliper event so I opted for adding it later.